iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/TapChangerStepImpl.java

```diff
@@ -30,12 +34,26 @@ protected TapChangerStepImpl(double rho, double r, double x, double g, double b)
         this.b = b;
     }
 
+    protected void setPosition(int position) {
+        this.position = position;
+    }
+
+    protected void setParent(AbstractTapChanger<?, ?, ?> parent) {
+        this.parent = parent;
+    }
+
+    protected void notifyUpdate(String attribute, Object oldValue, Object newValue) {
+        this.parent.getNetwork().getListeners().notifyUpdate(this.parent.parent.getTransformer(), () -> this.parent.getTapChangerAttribute() + ".step[" + position + "]." + attribute, oldValue, newValue);
```

Comment:
`this` is not necessary.
Maybe `stepHolder` or `tapChanger` is better than `parent`